python/activator/middleware_interface.py

```diff
@@ -282,20 +278,55 @@ def _prep_collections(self):
         #             'refcats'],
         #     output=self.output_collection)
         # The below is taken from SimplePipelineExecutor.prep_butler.
-        output_run = f"{self.output_collection}/{self.instrument.makeCollectionTimestamp()}"
+        self.output_run = f"{self.output_collection}/{self.instrument.makeCollectionTimestamp()}"
```

Comment:
If hanging onto this at the object level is part of the workaround for DM-34202, can you add a comment to that effect? I would have thought that you could just use `self.output_collection` once it was properly set up.